### ISSUE TITLE
Question on the update for meta learner

### ISSUE THREAD

**Issue by gzliyan113** [dragen1860/Reptile-Pytorch#2]
Hi, thanks for sharing the code!

I have a question on updating the meta learner: in the original paper, the update for meta learner is interpolation between the previous parameter and the parameters learned by the local learners.
However, it seems that here you are updating the meta learner based on the gradients produced by local learners on the query set. I was wondering is this implementation the same as the original paper's implementation?

Thanks!

**Comment by machanic** [comment]
@gzliyan113 I think this implementation is not Reptile, but the MAML. As you can see MAML's update is based on the sum of gradients over all tasks.

**Comment by HardworkingPearl** [comment]
@gzliyan113 Yes, I have the same doubt as you. I think the best way to modify this code to follow reptile paper is to add the gradients on both support set and query set.

**Comment by tingxueronghua** [comment]
I think so, too!

**Comment by chenmozxh** [comment]
agree with you!

**Comment by lzl19971215** [comment]
agree, I think this is the implementation of FOMAML.
